package.json

```diff
@@ -54,17 +54,16 @@
     "ts-node": "^8.0.1",
     "tslint": "^5.12.1",
     "typescript": "^3.2.4",
-    "zone.js": "^0.8.28"
+    "zone.js": "~0.8.28"
```

Comment:
What's driving this change?

Reply:
Our version of Angular (4-7) is looking for the `~0.8.6` range. See: https://github.com/angular/angular/issues/29570